pkg/cmdutils/errors.go

```diff
@@ -0,0 +1,37 @@
+package cmdutil
+
+import (
+	"errors"
+
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/viper"
+)
+
+// FlagError is the kind of error raised in flag processing
+type FlagError struct {
+	Err error
+}
+
+func (fe FlagError) Error() string {
+	return fe.Err.Error()
+}
+
+func (fe FlagError) Unwrap() error {
+	return fe.Err
+}
+
+// ErrorSilent is an error that triggers exit code 1 without any error messaging
+var ErrorSilent = errors.New("SilentError")
+
+// PrintError Prints in specified format
+func PrintError(err error, errString string) {
+	hasDebug := viper.GetBool("debug")
+
+	if err == ErrorSilent {
+		return
+	}
+	if hasDebug {
```

Comment:
I am not sure that PrintError should have the debug check. All log statement that would normally be in debug mode should be printed when the flag is set to true.

Reply:
debug check just enables `panic` statement, which is nothing but a traceback of error. Rest all logs will normally functions even without debug